opcpa_tpr_config/widgets.py

```diff
@@ -523,6 +582,38 @@ def db(self):
     def expert_mode(self):
         return self.expert_checkbox.isChecked()
 
+    @property
```

Comment:
Clever fix for using offset everywhere. I like it, although I feel like I always under utilize decorators.